### PR TITLE
In tutorials/quantize_vit, extract common methods to util.py

### PR DESCRIPTION
**Pull request by lancerts**
In tutorials/quantize_vit, extract common methods to util.py

Local execution tested via ./run.sh

cc @msaroufim

### CODE REVIEW

File: tutorials/quantize_vit/run_vit_b.py [hunk 1]
```diff
@@ -1,40 +1,18 @@
 import torch
 import torchvision.models.vision_transformer as models
 
+from util import benchmark_model, profiler_runner
+
```
Comment (msaroufim) [inline tutorials/quantize_vit/run_vit_b.py:5]:
```suggestion
torch.set_float32_matmul_precision("high")
```

File: tutorials/quantize_vit/run_vit_b_quant.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 import torchao
 import torchvision.models.vision_transformer as models
 
+from util import benchmark_model, profiler_runner
+
```
Comment (msaroufim) [inline tutorials/quantize_vit/run_vit_b_quant.py:6]:
```suggestion
torch.set_float32_matmul_precision("high")
```

File: tutorials/quantize_vit/util.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import torch
```
[the remark this answers is not in the document]
Comment (lancerts) [reply tutorials/quantize_vit/util.py:1]: Yeah, indeed not as convenient.

File: tutorials/quantize_vit/run_vit_b_quant.py (hunk 2 above)
Comment (lancerts) [reply tutorials/quantize_vit/run_vit_b_quant.py]: Just wondering, the tutorial is in `bf16`. 
```
# Set the model to evaluation mode
model.eval().cuda().to(torch.bfloat16)
```
Does the flag of float32 have any effect? 
Another question I'm curious about, we have `torch.set_float32_matmul_precision`
but it seems we don't have this for other dtype?

File: tutorials/quantize_vit/run_vit_b_quant.py [hunk 4]
```diff
@@ -19,30 +21,6 @@
 
 model = torch.compile(model, mode='max-autotune')
 
-def benchmark_model(model, num_runs, input_tensor):
```
[the remark this answers is not in the document]
Comment (lancerts) [reply tutorials/quantize_vit/run_vit_b_quant.py:22]: Checked the torchao codebase, there is no benchmark_model and profiler_runner currently. Moved them into the utils.py.

**Review by msaroufim — APPROVED** [review]

**Comment by lancerts** [comment]
```
 =================================== FAILURES ===================================
  _____ TestWeightOnlyInt8Quant.test_weight_only_quant_force_mixed_mm_4_cuda _____
  
  a = (<test_integration.TestWeightOnlyInt8Quant testMethod=test_weight_only_quant_force_mixed_mm_4_cuda>,)
  kw = {}
  
      @wraps(func)
      def standalone_func(*a, **kw):
  >       return func(*(a + p.args), **p.kwargs, **kw)
  
  /opt/conda/envs/venv/lib/python3.9/site-packages/parameterized/parameterized.py:620: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  /opt/conda/envs/venv/lib/python3.9/site-packages/torch/utils/_contextlib.py:115: in decorate_context
      return func(*args, **kwargs)
  test/integration/test_integration.py:1129: in test_weight_only_quant_force_mixed_mm
      self.assertTrue("mixed_mm" in code)
  E   AssertionError: False is not true
```
Not sure how this test failure relates to the code change....

**Comment by msaroufim** [comment]
It's unrelated, we'll merge the fix then your change so no worries!

**Comment by msaroufim** [comment]
@lancerts can you rebase to main? The error should go away

**Comment by lancerts** [comment]
> @lancerts can you rebase to main? The error should go away

Many thanks for the rebase... Just saw the message after work...

**Pull request merged**